### DISCUSSION TITLE
How to delete the default path alias?

### DISCUSSION THREAD

**Discussion by lihanspace** [nuxt/nuxt#16934] [Questions]
How to delete the default path alias?
Such as `~~`, `~`, `@@`

**Comment by TheAlexLichter** [comment]
Right now, this is not possible. You can only override them using `alias` in the `nuxt.config.ts` - but not fully remove them

**Comment by pi0** [comment]
Can you please explain why you need to delete these built-in aliases?

**Reply by lihanspace** [reply]
The autocomplete of vscode is `~~`, not `@` I want.

![image](https://user-images.githubusercontent.com/60546798/188785591-4f215cb0-ef6d-411d-b2cd-ca7ab03b7995.png)
